_pages/ChristianLastova.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: ChristianLastova
+pageUrl: ChristianLastova.github.io
+timestamp: 2017-01-25
+---
```

Comment:
@ChristianLastova just add a line break after line 5 and you should be all set 👍